### DISCUSSION TITLE
UI library for designer

### DISCUSSION THREAD

**Discussion by Giuly92** [wix/react-native-ui-lib#1855] [Q&A]
Hi everybody
Is it possible to find the library for Designer that is usable in Sketch or Figma?

**Comment by da1z** [comment]
> Hi everybody
> 
> Is it possible to find the library for Designer that is usable in Sketch or Figma?

Just bumping this question. Is there any plans to release figma components library of RNUILib ?

**Comment by ashish-tag** [comment]
Bump again

**Comment by d3thomas** [comment]
What is the status on this one? Would be much appreciated
